### DISCUSSION TITLE
Social network for Jamulus like JamKazam ?

### DISCUSSION THREAD

**Discussion by newlaurent62** [jamulussoftware/jamulus#2615] [Ideas]
Hello,

Has anyone think of an open source musical jamming social network like jamkazam for all internet musical jamming software ? 

Features like:
    - evaluate distance between you and a contact on the network
    - invite to a jam an online or offline contact
    - send/receive messages to your friends on/offline...

Jamulus is really a great tool, and can set place for encounters but when everybody is offline, we cannot contact the internet musicians we have met.

BR,
Laurent

**Comment by newlaurent62** [comment]
Or put an alert on a server to be notified when people connect to it

**Comment by hoffie** [comment]
That sounds like an interesting and really big feature. I think that 99% of it would be best developed as an external project (similar to Jamulus explorer). This allows choosing a nicely fitting technology stack (I guess it'll be a web application and C++ might not be the best fit there).
It will probably be helpful to have some kind of integration to the Jamulus client (e.g. URL handlers).

**Comment by reinhardwh** [comment]
This URL list all the musicians and when they have been online
https://jamulus.haggy.org

<img width="800" alt="grafik" src="https://user-images.githubusercontent.com/77500932/165308261-0292d9f7-6cf0-42f9-9fb4-f825b2d304f8.png">

**Reply by rdica** [reply]
Have you seen https://jamulus.live ?

**Comment by newlaurent62** [comment]
Same discussion on facebook Jamulus Official group [https://www.facebook.com/groups/jamulus/permalink/1122813385234397/](https://www.facebook.com/groups/jamulus/permalink/1122813385234397/)
